CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 > # Upcoming release
 >
 > ### Added
+- **UITableViewExtentions**:
+  - Added `isValidIndexPath(_ indexPath:)` method to check whether given IndexPath is valid within UITableView. [#441](https://github.com/SwifterSwift/SwifterSwift/pull/441) by [setoelkahfi](https://github.com/setoelkahfi).
 > ### Changed
 - **UITableViewExtentions**:
```

Comment:
@setoelkahfi, could you please add your changes here, we don't have to create a new **UITableViewExtentions** title if it already exists

Reply:
Oh, so it should be in the 'Changed' section of UITableViewExtentions?

Reply:
Oops, sorry my bad @setoelkahfi I didn't notice it was under changed 😅